core/iwasm/interpreter/wasm_loader.c

```diff
@@ -7128,6 +7128,30 @@ check_branch_block(WASMLoaderContext *loader_ctx, uint8 **p_buf, uint8 *buf_end,
     return NULL;
 }
 
+#if WASM_ENABLE_EXCE_HANDLING != 0
+static BranchBlock *
+check_branch_block_for_delegate(WASMLoaderContext *loader_ctx, uint8 **p_buf,
+                                uint8 *buf_end, char *error_buf,
+                                uint32 error_buf_size)
+{
+    uint8 *p = *p_buf, *p_end = buf_end;
+    BranchBlock *frame_csp_tmp;
+    uint32 depth;
+
+    read_leb_uint32(p, p_end, depth);
+    CHECK_BR(depth);
```

Comment:
Not sure whether it should be `CHECK_BR(depth + 1)`? Seems the target block to check is `loader_ctx->frame_csp - depth - 2`.
In check_branch_block, the code is:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/3111a86cb341cee9eb0b3e7d8d1d43327846a627/core/iwasm/interpreter/wasm_loader.c#L7120-L7121

Reply:
good point. i will check.

Reply:
fixed.